datasets/co/funcion_publica/co_funcion_publica.yml

```diff
@@ -0,0 +1,125 @@
+title: Colombian Administrative Department of Public Service PEP Declarations
+entry_point: crawler.py
+prefix: co-cedula
```

Comment:
nope, then this applies to positions etc too

Reply:
Aaaargh. Sometimes a simple string function is better than the framework :)